tests/cmd/main_test.py

```diff
@@ -49,21 +49,21 @@ def _read_output_file():
 
 
 def test_single_file():
-    args = ["undebt", "-i", method_to_function_input_path, "-p", method_to_function_path, "--verbose"]
+    args = ["undebt", "-p", method_to_function_path, "--verbose", method_to_function_input_path]
```

Comment:
Can you change this and the other tests to move the `--verbose` to the end? I'd like to make sure that if the arguments are of the form `undebt -p pattern input` that `input` isn't treated as another pattern, but this test doesn't check that, since the `--verbose` is in the way.